doc/integrator/docker.rst

```diff
@@ -87,34 +87,36 @@ Geoportal:
  * ``AUTHTKT_SAMESITE``: See: :ref:`integrator_authentication`.
  * ``BASICAUTH``: See: :ref:`integrator_authentication`.
  * ``LOG_TYPE``: Should be 'console' with Docker Compose and 'json' with OpenShift.
- * ``LOG_LEVEL``: Log level for your application, default is ``INFO``, can be
+ * ``LOG_LEVEL``: Log level for your application, default to ``INFO``, can be
```

Comment:
This change is grammatically incorrect, either "default is", or "defaults to"
Same for the following

Reply:
Then this https://ludwig.guru/s/default+to+something is wrong, or I don't understand something...

Reply:
The example https://ludwig.guru/s/default+to+something contains a verb within the formulation "it'll" (correct language would be "it will"), so it's a different structure than our sentence.

If we want to use "to" like in the example from ludwig, we would also need a verb, for example
default is set to
or
defaults to